app/components/buttons/DeleteButton.tsx

```diff
@@ -11,17 +12,18 @@ export default function DeleteButton({
   active: boolean;
 }) {
   return (
-    <button
+    <Button
+      color="danger"
       disabled={!canDelete}
       type="button"
       onClick={() => setShowModal(true)}
       className={classNames(
-        'group flex items-center px-4 py-2 text-sm w-full',
+        'group flex items-center px-4 py-2 text-sm w-full transition duration-300 ease-in-out',
```

Comment:
1. Let's use the `size` prop of the Button component to set its size.
2. Can we try removing all classes to see if they're actually necessary?